cli/worker.rs

```diff
@@ -596,6 +596,7 @@ impl CliMainWorkerFactory {
         node_ipc_fd: shared.node_ipc,
         disable_deprecated_api_warning: shared.disable_deprecated_api_warning,
         verbose_deprecated_api_warning: shared.verbose_deprecated_api_warning,
+        future: std::env::var("DENO_FUTURE").is_ok(),
```

Comment:
This env var should be captured in `cli/args/mod.rs` and exposed as a method on `CliOptions` struct - maybe `CliOptions::enable_future_features()`?